### PR TITLE
[framework-bundle] Disable the logger in test env

### PR DESCRIPTION
**Pull request by dunglas**
| Q             | A
| ------------- | ---
| License       | MIT

### CODE REVIEW

File: symfony/framework-bundle/3.3/config/packages/test/framework.yaml [hunk 1]
```diff
@@ -2,3 +2,7 @@ framework:
     test: ~
     #session:
     #    storage_id: session.storage.mock_file
+
+services:
+    # Disable logs
```
Comment (ogizanagi) [inline symfony/framework-bundle/3.3/config/packages/test/framework.yaml]: As soon as Monolog is used with a proper stream handler for instance, registering this null logger probably isn't a good idea.

Comment (dunglas) [reply symfony/framework-bundle/3.3/config/packages/test/framework.yaml]: Why? It will speed up the builds, even with Monolog.

Comment (ogizanagi) [reply symfony/framework-bundle/3.3/config/packages/test/framework.yaml]: Logs can be useful to understand why a test fail. I'm not sure about this as is in the recipe.

Comment (dunglas) [reply symfony/framework-bundle/3.3/config/packages/test/framework.yaml]: Anyway monolog has no specific config for tests right now.
It's just a recipe, for specific needs, debugging etc, the user can (should?) modify this file to fit his needs.

Comment (stof) [reply symfony/framework-bundle/3.3/config/packages/test/framework.yaml]: the question then is whether this logger is winning over monolog or no. This should be checked
The goal is to have it winning over the default logger added in FrameworkBundle when no logger is defined

**Comment by fabpot** [comment]
closing in favor of #281

File: symfony/framework-bundle/3.3/config/packages/test/framework.yaml (hunk 1 above)
Comment (lsmith77) [reply symfony/framework-bundle/3.3/config/packages/test/framework.yaml]: or maybe we should make it possible to disable the default logger via config

**Pull request closed without merging**